tensorflow/python/eager/function.py

```diff
@@ -3191,10 +3199,16 @@ def _create_graph_function(self, args, kwargs, override_flat_arg_shapes=None):
         shared_func_graph=False)
     return graph_function
 
-  def _define_function_with_shape_relaxation(self, args, kwargs):
+  def _define_function_with_shape_relaxation(self,
+                                             args,
+                                             kwargs,
+                                             flat_args,
+                                             flat_kwargs):
```

Comment:
is this still needed?

Reply:
It's not absolutely required, but there is still a call to `nest.flatten` with expanded composites in L3247/3261 that it replaces